components/arrow/package.json

```diff
@@ -17,10 +17,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```

Comment:
not sure who wrote this - probably best to leave it neutral as "The govuk-frederic authors"